services/collaboration/pkg/config/defaults/defaultconfig.go

```diff
@@ -44,12 +40,13 @@ func DefaultConfig() *config.Config {
 			Zpages: false,
 		},
 		WopiApp: config.WopiApp{
-			Addr:     "https://127.0.0.1:8080",
+			Addr:     "https://127.0.0.1:9980",
 			Insecure: false,
+			WopiSrc:  "https://localhost:9300",
```

Comment:
if the collaboration service is started standalone it is reachable on localhost. Config doc string and README give an example when running on a different domain.